### DISCUSSION TITLE
wurst mod crashing

### DISCUSSION THREAD

**Discussion by JAfferte** [FabricMC/fabric-api#1524] [Player support]
Mod 'Wurst Client' (wurst) requires version 0.33.4 or later of mod fabric, which is missing!
You must install version 0.33.4 or later of fabric.
even though i have the 1.17 api

**Comment by Martmists-GH** [comment]
We do not support hacked clients, cheats, or modifications that are intended to give you the upper hand in a server, competitive or otherwise.

**Comment by TheAwesome98-Real** [comment]
you need to install a newer version of the fabric api
